pandas/_libs/algos_rank_helper.pxi.in

```diff
@@ -54,7 +58,7 @@ def rank_1d_{{dtype}}(object in_arr, ties_method='average', ascending=True,
         {{if dtype == 'uint64'}}
         {{ctype}} val
         {{else}}
-        {{ctype}} val, nan_value
+        {{ctype}} val, nan_value, isnan
```

Comment:
isnan is a very confusing name here as this is an actual value

Reply:
should I just call it masked ?